Basic/operator/README.md

```diff
@@ -86,4 +85,21 @@ berikut tabel dari operasi perbandingan
 |      >        |           Lebih besar dari            |
 |      <        |           Lebih Kecil dari            |
 |      >=       |    Lebih Besar dari atau sama dengan  |
-|      <=       |    Lebih Kecil dari atau sama dengan  |
\ No newline at end of file
+|      <=       |    Lebih Kecil dari atau sama dengan  |
+
+## Operasi Logika
+operasi logika adalah operasi yang digunakan untuk mengecek pernyataan apakah True atau False
+
+sebagai contoh operasi logika
+```python
+print(3 < 5 and 5 > 3) # True
+```
+
+operasi tersebut mengecek apakah kedua nilai yang di cek memiliki nilai True
+
+berikut tabel dari operasi logika
+|   Operator    |                         Penjelasan                                    |
+|---------------|-----------------------------------------------------------------------|
+|     and       | Mengembalikan nilai True jika kedua nilai memiliki nilai True         |
+|     or        | Mengembalikan nilai True jika salah satu nilai memiliki nilai True    |
+|     not       | Membalikkan hasil, mengembalikan False jika hasilnya True             |
```

Comment:
Ini agak kurang tepat sih.

`and` -> mengembalikan nilai *falsy* yang pertama ditemukan. Misalnya: `1 and 0` dan `0 and 1` keduanya bakal ngembaliin 0. `None and 1` bakal ngembaliin `None`. Kalau gaada nilai yang *falsy*, maka operan paling kanan yang dikembaliin, misalnya: `1 and 2` bakal ngembaliin 2.

`or` -> mengembalikan nilai *truthy* pertama yang ditemukan. Misalnya: `1 or 2` bakal return 1. `0 or 2` bakal return 2.

`not` -> mengembalikan `True` jika nilainya *falsy* atau sebaliknya, mengembalikan `False` jika nilainya *truthy*.